state/protocol/badger/mutator.go

```diff
@@ -774,6 +778,9 @@ func (m *FollowerState) handleServiceEvents(block *flow.Block) ([]func(*transact
 					return nil, state.NewInvalidExtensionErrorf("invalid epoch commit: %s", err)
 				}
 
+				// update `committed_epoch_final_view` metric with new final view
+				m.complianceMetrics.CommittedEpochFinalView(setup.FinalView)
```

Comment:
```suggestion
				m.metrics.CommittedEpochFinalView(setup.FinalView)
```
To access the embedded metrics in `State`